src/pages/STG_SettingsManager.page

```diff
@@ -80,161 +76,146 @@
-        z-index: 99;
-        position: fixed;
-        top: 30%;
-        left: 30%;
-        background: lightyellow;
-        width: 400px;
-        height: 40px; 
-        border-radius: 4px;      
-    }   
-
-    /* END - COMMON CSS FOR ALL SETTINGS PAGES */
-    
-    /* SIDEBAR CSS */
-        
+
+
+    /* END - COMMON CSS FOR ALL SETTINGS PAGES *
+
+    /* SIDEBAR CSS *
+
     .sidebar ul {
-        margin: 0; 
-        padding: 0; 
+        margin: 0;
+        padding: 0;
         list-style: none;
-        background: lightgray; 
+        background: lightgray;
     }
-        
-    div.navbar ul li {
-        font-size: 100%; 
+
+    div.slds-tree-container ul li {
+        font-size: 100%;
         margin-bottom: 0px;
         margin-left: 0px;
-        padding: 2px; 
+        padding: 2px;
         background: #f2f5f6;
-        font-weight: normal; 
+        font-weight: normal;
         border: 1px solid #dce4e7;
         border-top-color: #fff;
     }
-        
-    div.navbar ul li:hover {
-        background: gray; 
+
+    div.slds-tree-container ul li:hover {
+        background: gray;
     }
-    
-    div.navbar ul li.subitem:hover {
-        background: gray; 
+
+    div.slds-tree-container ul li.slds-tree__item:hover {
+        background: gray;
     }
-    
-    div.navbar ul li.subitem {
-        background: #eaeef0; 
+
+    div.slds-tree-container ul li.slds-tree__item {
+        background: #eaeef0;
         border-bottom: 0;
-        border-top: 0;        
+        border-top: 0;
     }
-    
-    div.navbar {
+
+    div.slds-tree-container {
        border-top: 1px solid #dce4e7;
     }
-    
-    div.navbar ul li a {
-        display: block; 
-        text-decoration: none; 
+
+    div.slds-tree-container ul li a {
+        display: block;
+        text-decoration: none;
         padding: 5px 5px 5px 10px;
-        font-size: 100%; 
+        font-size: 100%;
         color: black;
     }
-        
-    div.navbar ul li a:hover {
-        color: #fff; 
-        background: none; 
-    }
-    
-    div.navbar ul li.nav-selected {
-        background: LightGray;
+
+    div.slds-tree-container ul li a:hover {
+        color: #fff;
+        background: none;
     }
 
+
+
     .nav-section-title {
-        display: block; 
-        text-decoration: none; 
+        display: block;
+        text-decoration: none;
         padding: 5px 5px 5px 10px;
-        font-size: 100%; 
+        font-size: 100%;
         color: black;
     }
-        
-    li.collapsed .caret {
-        float: right;
-        width: 0;
-        height: 0;
-        display: inline-block;
-        vertical-align: top;
-        content: "";
-        margin-top: 8px;
-        margin-left: 2px;
-        border-top: 4px solid #000;
-        border-bottom: 0px;
-        border-right: 4px solid transparent;
-        border-left: 4px solid transparent;
-    }
-
-    li .caret {
-        float: right;
-        width: 0;
-        height: 0;
+        */
```

Comment:
it looks like this whole upper section of CSS terminating on this line is commented out.  If so, shall we remove it?